bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/cloud/CloudConnector.java

```diff
@@ -137,14 +137,22 @@ public boolean isConnected() {
             mapCountry = country.trim().toLowerCase();
             mapUrl = cl.getMapUrl(mapId, mapCountry);
         }
-        @Nullable
-        RawType mapData = HttpUtil.downloadData(mapUrl, null, false, -1);
-        if (mapData != null) {
-            return mapData;
-        } else {
-            logger.debug("Could not download '{}'", mapUrl);
+        if (mapUrl.isEmpty()) {
+            logger.debug("Cannot download map data: Returned map URL is empty");
             return null;
         }
+        try {
+            RawType mapData = HttpUtil.downloadData(mapUrl, null, false, -1);
+            if (mapData != null) {
+                return mapData;
+            } else {
+                logger.debug("Could not download '{}'", mapUrl);
+                return null;
+            }
+        } catch (IllegalArgumentException e) {
+            logger.debug("Error downloading map: {}", e.getMessage());
```

Comment:
Exceptions should be logged as the placeholder-less last parameter.

Reply:
thanks @t-8ch for your comment.
not sure that I grasp what you mean with your comment.

Do you mean like ` logger.debug("Error downloading map: ", e); ` ?  my understanding is that gives the full stacktrace rather than just the message, which is not what I want. Purpose of this PR is to get rid of the full error /stacktrace in folks log.